underscore.js

```diff
@@ -1287,11 +1292,7 @@
 
   _.noop = function(){};
 
-  _.property = function(key) {
-    return function(obj) {
-      return obj == null ? void 0 : obj[key];
-    };
-  };
+  _.property = property;
```

Comment:
To keep it inline with the other Object functions. Same as with the `create*` helper functions, which we then [assign to methods](https://github.com/jashkenas/underscore/blob/master/underscore.js#L1001-L1005).